app/views/ExposureHistory/NextSteps.tsx

```diff
@@ -12,21 +13,23 @@ import { AUTHORITY_NAME as healthAuthority } from '../../constants/authorities';
 
 const NextSteps = (): JSX.Element => {
   const navigation = useNavigation();
+  const { t } = useTranslation();
   useStatusBarEffect('light-content');
 
   const handleOnBackPress = () => {
     navigation.goBack();
   };
 
-  const headerText = `${healthAuthority} recommends you take a self-assessment`;
-
-  const contentTextOne =
-    'It is possible that you may have crossed paths with somebody who has been diagnosed with COVID-19.';
-
-  const contentTextTwo =
-    "This does not mean that you are infected, but you should take precautions anyway. People who don't exhibit symptoms can sometimes still be contagious.";
-
-  const buttonText = 'Take Self Assessment';
+  const headerText = t('exposure_history.next_steps.ha_self_assessment', {
+    healthAuthority,
```

Comment:
This was already here, but i might consider using `healthAuthorityName` here instead.

And as a side note worth mentioning the concept of a health authority is somewhat different between GPS and BT at this point, i see the team needing to work through this distinction coming up soon. This HA here seems to be more of the BT flavor of the concept, just so you know.

Reply:
thanks, I've updated the naming, but I see that the name of the HA is static (`Boston Public Health Commission`).
Shouldn't we pass it as a prop or add it to the context?

Reply:
One note on the HA part -- We'll be gating that off for GPS for now, I believe that's just for testing?